samples/apps/copilot-chat-app/webapp/tests/chat.test.ts

```diff
@@ -1,15 +1,42 @@
+/* eslint-disable testing-library/prefer-screen-queries */
```

Comment:
nit: breaking the tests into multiple files, i.e chat, plugin, multi-user, basic-ui-interaction, etc.

Reply:
I mentioned this in the description but when there are multiple simultaneous async tests running, they interfere with each other, by randomly sending messages to the chat server (all the tests share the same chat server), and this confuses the LLM. So a lot of the checks for chat history has n messages or last message was by copilot fail. Sometimes plan execution is also just abandoned cause the LLM thinks we've moved on.
The execution of all of these tests also requires knowing the exact state of the webpage, since without that locators wont work. and when tests start randomly after each other this cant be guaranteed.

Reply:
One thing we can do is to create a new chat session for each test.